api/src/tests/staticLayers.js

```diff
@@ -13,13 +13,6 @@ jest.mock('../config/config.js', () => {
 });
 
 describe('GET /api/staticLayers', () => {
-  it('should return empty array, if no geojson data in db', async () => {
```

Comment:
please don't remove test, but modify it. In the PR you wrote quite nice scenario and this one test needs only change in description and on L21 to expect the one layer